Apps/MineOS/docker-compose.yml

```diff
@@ -0,0 +1,162 @@
+name: mineos-node
+# Base file from: https://github.com/hexparrot/mineos-node/blob/master/docker-compose.yml
+
+services:
+  mineos:
+    image: hexparrot/mineos:latest
+    container_name: mineos_server
+    networks:
+      - mineos
+    restart: unless-stopped
+
+    # Adjust as needed; if this is too short, stopping the container
+    # may corrupt map data
+    stop_grace_period: 10m
+
+    volumes:
+      # Using a bind mapping is preferrable here due to how docker
+      # stores names volumes by default in /var/lib/docker/volumes
+      # with permissions that prevent normal users from accessing
+      # the folder structure.
+      - /var/games/mineos/minecraft:/var/games/minecraft
+
+    environment:
+      # Should the web interface use HTTPS or HTTP?
+      # Valid entries are "true" and "false" (with quotes)
+      USE_HTTPS: "true"
+
+      # What port should the web interface listen on?
+      # This is independant of the USE_HTTPS setting.
+      SERVER_PORT: 8443
+
+      # On startup, mineos will check if an account of this name exists.
+      # If it does not exist, one will be created. This account, and
+      # any other created with this variable will be permitted to login
+      # to the web interface.
+      USER_NAME: mc
+
+      # If the account listed above does not already exist, when mineos
+      # creates it, it will do so with UID. If an account with this UID
+      # already exists, account creation will be aborted.
+      USER_UID: 1000
+
+      # If the account is being created, it will use either the password
+      # set below, or a random, 20 character password will generated for
+      # you and displayed in the console log. After first launch, you
+      # should check the log.
+      USER_PASSWORD: random_see_log
+
+    ports:
+      # HTTPS port for web interface. If you change the SERVER_PORT
+      # variable, also change this mapping.
+      - "8443:8443/tcp"
+
+      # Minecraft server ports. 25565 is the default, but is you plan
+      # on running multiple servers at the same time, each will need
+      # a unique port unless you are using bungiecord.
+      # Expand this range as needed.
+      - "25565-25570:25565-25570/tcp"
+      - "25565-25570:25565-25570/udp"
+
+    x-casaos:
+      envs:
+        - container: TZ
+          description:
+            en_us: TimeZone
```

Comment:
for example this is a description for environment variable `TZ`.

But it is not required - it's completely up to you.